.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./node_modules/gts"
+  "extends": "./node_modules/gts",
+  "rules": {
+    "no-unused-vars": [{ "argsIgnorePattern": "^_"}]
+  }
```

Comment:
is this ok? it would get rid of some `// eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
oops, guess not :-(

Reply:
actually, this made `gts` happy: `"@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_"}]`